js/ccf-app/package.json

```diff
@@ -23,12 +23,12 @@
     "@types/node": "17.0.8",
     "@types/node-forge": "^0.10.9",
     "chai": "^4.3.4",
+    "colors": "^1.4.0",
```

Comment:
No!

Reply:
Well spotted! Hmm, my VSCode seems to automatically update this file. Tracking down which extension is responsible for this! In the meantime, I've reverted this change.